packages/widgets/src/components/NotificationMenu/NotificationMenu.tsx

```diff
@@ -0,0 +1,78 @@
+import { useTheme } from '@emotion/react';
+import {
+  Icon,
+  Popover,
+  PopoverClose,
+  PopoverContent,
+  PopoverDescription,
+  PopoverHeading,
+  PopoverTrigger,
+} from '@ssa-ui-kit/core';
+import { NotificationMenuProps } from './types';
+import {
+  ResetBtnStyles,
+  Close,
+  ButtonsWrapper,
+  List,
+  Loading,
+  PopoverWrapper,
+  PopoverContentWrapper,
+} from './styles';
+import { NotificationCard } from '@components/NotificationCard';
+
+export const NotificationMenu = ({
+  trigger,
+  notifications,
+  children,
+  onClick,
+  isLoading,
+  rightButton,
+  leftButton,
+}: NotificationMenuProps) => {
+  const theme = useTheme();
+  return (
+    <Popover placement="bottom-end">
+      <PopoverTrigger onClick={onClick} css={ResetBtnStyles}>
+        {trigger}
+      </PopoverTrigger>
+      <PopoverContent className="popover" css={PopoverWrapper}>
+        <div css={PopoverContentWrapper}>
+          <PopoverClose css={Close}>
+            <Icon name="cross" size={14} color={theme.colors.greyGraphite70} />
+          </PopoverClose>
+          <PopoverHeading variant="h5" weight="bold">
+            Notification
+          </PopoverHeading>
+          {isLoading ? (
+            <div css={Loading}></div>
+          ) : (
+            <PopoverDescription variant="body1">
+              <div data-testid="button-group">{children}</div>
```

Comment:
I would call it differently without specifying that we expect exactly a button group here. 
A user can pass here whatever they want.

Reply:
Done